app/views/notifications/index.erb

```diff
@@ -56,71 +56,81 @@
           <% end %>
         </ul>
       <% end %>
-
-      <ul style="list-style-type: none; margin: 0;">
-        <% inbox.notifications.each do |notification| %>
-          <li class="alert-row">
-            <span style="padding-right: 15px;" class="fa fa-<%= notification.icon %>"></span>
-
-            <% if notification.action == 'like' %>
-              <a href="/<%= notification.actor.username %>">
+      <% notification_hash = Hash.new([]) %>
+      <% inbox.notifications.each do |notification| %>
```

Comment:
Any issue with me making it a method on inbox? I'm thinking notifications_by_language